aio/content/guide/template-statements.md

```diff
@@ -18,7 +18,7 @@ Una declaración de plantilla *tiene un efecto secundario*.
 Ese es el objetivo de un evento.
 Es la forma de actualizar el estado de la aplicación a partir de la acción del usuario.
 
-Responder a los eventos es el otro lado del "flujo de datos unidireccional" de Angular.
+Responder a los eventos es el otro lado del `flujo de datos unidireccionales` de Angular.
```

Comment:
Hola, gracias por el aporte @gustavguez 🚀  ¿Podríamos considerar algo así para mantener el sentido de la oración original en inglés? .

También estaba viendo que le falta el _div_ con la clase _alert is-helpful_ de acuerdo al [archivo original ](https://github.com/angular/angular/edit/master/aio/content/guide/template-statements.md).

```suggestion

<div class="alert is-helpful">

Responder a los eventos es un aspecto del `flujo unidireccional de datos` de Angular.

</div>
```